ecs/README.md

```diff
@@ -2,7 +2,11 @@
 
 Create a compute cluster for hosting docker based apps.
 
+## Notes
 
+The `idle_timeout` is set to 60 seconds. Make sure that your webserver's keep-alive timeout is set to 60 seconds (or more) to prevent the load balancer from keeping connections open to that the server has already closed.
```

Comment:
```suggestion
The `idle_timeout` is set to 60 seconds. Ensure that your webserver's keep-alive timeout is set to 60 seconds (or more) to prevent the load balancer from keeping connections open to the already closed server.
```